lib/Reader.php

```diff
@@ -130,7 +130,7 @@ public function parseGetElements(array $elementMap = null): array
      *
      * @param array<string, mixed>|null $elementMap
      *
-     * @return array<string, mixed>|string|null
+     * @return array<int,array<string, mixed>>|string|null
```

Comment:
This was an existing error in the PHPdoc. This thing returns an array of arrays.